maxim/apis/maxim_apis.py

```
@@ -1651,6 +1666,247 @@ def execute_prompt_chain_for_data(
+        test_run_entry_id: str,
+    ) -> Dict[str, Any]:
+        """Poll simulation prompt status (GET). Returns dict with 'status' and optionally 'outputs' etc. when complete."""
+        try:
+            endpoint = (
+                f"/api/sdk/v2/test-run/execute/simulation/prompt"
+                f"?workspaceId={workspace_id}&testRunEntryId={test_run_entry_id}"
+            )
+            res = self.__make_network_call(method="GET", endpoint=endpoint)
+            json_response = json.loads(res.decode())
+            if "error" in json_response:
+                raise Exception(json_response["error"])
```

Ruff (0.14.14)

Abstract `raise` to an inner function

(TRY301)